src/app/redux/Login/reducers.js

```diff
@@ -3,14 +3,17 @@ import Immutable from 'seamless-immutable';
 import { actions } from './actions';
 
 const initialState = {
-  token: ''
+  isLoading: false
 };
 
 function reducer(state = Immutable(initialState), action) {
   switch (action.type) {
+    case actions.USER_LOGIN_LOADING:
+      return state.merge({ isLoading: true });
     case actions.USER_LOGIN_SUCCESS: {
       return state.merge({
-        token: action.payload.token
+        token: action.payload.token,
+        isLoading: false
```

Comment:
add `USER_LOGIN_FAILURE` to put isLoading in `false`

also, why to you put the `token` in the state?